unison-src/transcripts/move-namespace.md

```diff
@@ -1,9 +1,47 @@
 # Tests for `move.namespace`
 
+
+## Moving the Root
```

Comment:
I moved the global root stuff to the top of this transcript, leaving it at the bottom meant it would have a bunch of non-deterministic `.__projects.*` junk in it.

I'll end up rewriting this part in the project-roots branch.